src/Raven.Studio/typescript/commands/wizard/continueClusterConfigurationCommand.ts

```diff
@@ -14,7 +14,7 @@ class continueClusterConfigurationCommand extends commandBase {
         const url = endpoints.global.setup.setupContinue + this.urlEncodeArgs(args);
 
         return this.post<operationIdDto>(url, JSON.stringify(this.dto), null, { dataType: undefined })
-            .fail((response: JQueryXHR) => this.reportError("Failed to configure cluster node", response.responseText, response.statusText));
+            .fail((response: JQueryXHR) => this.reportError("Failed to configure cluster node in Secure mode", response.responseText, response.statusText));
```

Comment:
do we use this command in unsecured mode as well?

Reply:
+ I'd remove `in Secure mode`. This might imply that secure mode is the cause.

Reply:
done